src/assets/css/global/_main.scss

```diff
@@ -618,13 +618,31 @@ km-cni-version {
 }
 
 .km-quota-widget {
-  .km-quota-item {
+  km-property {
+    width: 100%;
+
     .container {
-      margin: 2px 5px;
+      width: 100%;
+
+      &:not(last-child) {
+        margin: 2px 5px;
+      }
     }
   }
 
   .mat-progress-bar .mat-progress-bar-fill {
     transition: none;
   }
+
+  .km-extended-widget {
```

Comment:
This style looks redundant because this is already handled by `km-quota-widget` style above. Or am I not understanding it right?